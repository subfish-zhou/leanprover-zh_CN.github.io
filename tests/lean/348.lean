def doSomething (db:String) : String := do
  let mut cl : Nat × Nat → Nat × Nat := λx => x
  cl := (λ(x,y) => (x, y⟩)
  db
