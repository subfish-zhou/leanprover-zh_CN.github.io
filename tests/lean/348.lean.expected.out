348.lean:3:24: error: expected ')'
